frontend/packages/erd-core/src/schemas/hash/types.ts

```diff
@@ -0,0 +1,4 @@
+import type { InferOutput } from 'valibot'
+import type { hashSchema } from './schemas'
+
+export type Hash = InferOutput<typeof hashSchema>
```

Comment:
This `Hash` type is same with `${string}__columns__${string}`. It will extend its definition when we allow other hash values by updating `hashSchema`.